src/plugins/action_generate_review_assignment_records/src/databaseMethods.ts

```diff
@@ -0,0 +1,131 @@
+const databaseMethods = (DBConnect: any) => ({
+  getPersonnelForApplicationStageLevel: async (
+    templateId: number,
+    stageNumber: number,
+    reviewLevel: number,
+    type: 'Review' | 'Assign'
+  ) => {
+    const text = `
+    SELECT
+      "userId", "orgId",
+      "templatePermissionRestrictions" AS restrictions
+      FROM permissions_all
+      WHERE "templateId" = $1
+      AND "stageNumber" = $2
+      AND "reviewLevel" = $3
+      AND "permissionType" = '${type}'
+    `
+    try {
+      const result = await DBConnect.query({
+        text,
+        values: [templateId, stageNumber, reviewLevel],
+      })
+      return result.rows
+    } catch (err) {
+      console.log(err.message)
+      throw err
+    }
+  },
+  addReviewAssignments: async (reviewAssignments: any) => {
```

Comment:
would assume nothing changed in this method ?

Reply:
Nope, just copy-pasted.